client/src/test/java/io/pravega/client/stream/ReaderGroupConfigTest.java

```diff
@@ -76,6 +84,38 @@ public void testStartFromStreamCuts() {
         assertEquals(getStreamCut("s2"), cfg.getStartingStreamCuts().get(Stream.of("scope/s2")));
     }
 
+    @Test
```

Comment:
I don't see changes to `ReaderGroupConfig` in this PR. It sounds like you had changes to that class, but they were removed in recent commits. Why are we changing test code in `ReaderGroupConfigTest` then?

Reply:
I made changes to the `ReaderGroupConfig` for #5114 to use but those `ReaderGroupConfig` changes have already been added through #5114 . These tests are from my changes to the `ReaderGroupConfig`. I can remove them since they are obsolete now.